_assignments/mp4.md

```diff
@@ -0,0 +1,192 @@
+
+
+### Start Menu
+
+The player should start out in a separate scene, which will contain a start menu and tutorial. The start menu will be controlled via a laser pointer 
+placed in the scene. The start menu will require the use of the Oculus touch controllers. Both [Unity](https://docs.unity3d.com/Manual/OculusControllers.html) and [Oculus](https://developer.oculus.com/documentation/unity/latest/concepts/unity-ovrinput/) have pages dedicated to 
+touch controller input in Unity.
+
+
+1. Create a new scene, separate from your flight simulator, and a room within that scene. The room can be as simple or complex as you like, 
+   but it should be well-lit, and the walls should be a solid color.
+
+2. Somewhere in this room, place a start menu. At the very minimum, your menu must include the option to quit and to start the game.
+
+3. Upon selecting quit, the game should exit. Upon selecting start, the player should be transported to your flight simulator scene.
+
+4. We have provided a model for a pointing device. The start menu MUST be controlled by picking up this device with the touch controllers, 
+   and pointing it at the menu options. The player should be able to select the option being pointed at by pressing the A button. You MAY NOT 
+   control the menu with the touch controller joysticks. Hint: [Ray Casting](https://unity3d.com/learn/tutorials/modules/beginner/physics/raycasting) is a useful technique, and there are online resources for figuring 
+   out Touch input, such as the [Unity manual](https://docs.unity3d.com/Manual/OculusControllers.html) and the [Oculus developer guide](https://developer.oculus.com/documentation/unity/latest/concepts/unity-ovrinput/).
+
+
+### Flight
+
+Your plane must be comfortable, easy to control, and should feel like a real plane. For instance, real planes cannot turn in place, and cannot 
+stay in the air while standing still.
+
+1. You must use the touch controllers to control the plane. You can implement multiple types of controls during your development, but we will 
+   only check the version with the touch controllers. 
+
+2. Your plane must be able to accelerate/decelerate, and yaw, pitch, and roll. You should implement these functions in your own way with 
+   the touch controllers and Oculus Rift. 
+
+3. You MUST use the rotation and/or position tracking of the touch controllers in your control scheme somehow. Experiment with different 
+   configurations to figure out a comfortable and intuitive setup. Remember to consult the Oculus Best Practices Guide for suggestions. 
+
+---
+
+## Part 2 - Make it a Game!
+
+### Shooting Game
+
+You are now going to take your flight simulator, and turn it into a shooting game.
+
+1. Add a weapon, either a laser or physical bullets/rockets, to your plane. You decide from where the ray/bullets emanate from the plane. 
+   Do not use Unity’s Debug.DrawLine for your laser, as it will not appear in a built executable.

```

Comment:
```suggestion
   Do not use Unity’s `Debug.DrawLine` for your laser, as it will not appear in a built executable.
```
Programming names should use monospace